foundations/src/telemetry/log/init.rs

```diff
@@ -95,12 +97,29 @@ pub(crate) fn init(service_info: &ServiceInfo, settings: &LoggingSettings) -> Bo
         }
         (LogOutput::File(file), LogFormat::Json) => {
             let buf = BufWriter::with_capacity(BUF_SIZE, File::create(file)?);
-            let drain = build_json_log_drain(buf);
+            let drain = build_json_log_drain(buf, false);
             AsyncDrain::new(drain).chan_size(CHANNEL_SIZE).build()
         }
+        (LogOutput::Stdout, log_fmt) => {
+            let stdout = unsafe { File::from_raw_fd(STDOUT_FILENO) };
+            let buf = BufWriter::with_capacity(BUF_SIZE, stdout);
+
+            match log_fmt {
```

Comment:
I wanted to reduce the duplication a bit here by creating `stdout` in one place, but if the nested match is not the preferred style then I'm fine with keeping the pattern consistent with the other branches.